client/src/components/overview/overview-components/Carousel.jsx

```diff
@@ -1,22 +1,157 @@
 import React, { useState } from "react";
-import "react-responsive-carousel/lib/styles/carousel.min.css"; // requires a loader
 import { Carousel } from 'react-responsive-carousel';
 import styled from 'styled-components';
-
+import { findAverageRating } from '../GalleryView.jsx'
 
 const StyledOverviewCarousel = styled.div`
-grid-area: OvPicture;
+  grid-area: OvPicture;
+  max-width: 200vw;
+  display: flex;
+  align-items: center;
+  position: relative;
+  overflow: hidden;
+  border-radius: 2px;
+  background-color: white;
 `;
 
+const ThumbnailSelector = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-between;
+  width: 4%;
+  position: absolute;
+  width: auto;
+  height: 60%;
+  background-color: rgba(255,255,255,0.5);
+  overflow: auto;
+  direction: rtl;
+  overflow-x: hidden;
+  border-radius: 2px;
+  &::-webkit-scrollbar {
+    background: transparent;
+    cursor: pointer;
+  };
+  &::-webkit-scrollbar-thumb {
+    background: #cecece;
+    border-radius: 2px;
+  }
+`
+
+const Thumbnail = styled.div`
+  margin: 10px;
+  margin-top: 20px;
+  border: 2px solid grey;
+  width: 100px;
+  border-radius: 2px;
+  cursor: pointer;
+  transition: 0.3s;
+  &:hover {
+    border: 2px solid white;
+  }
+`
+
+// each photo has this class
+const Slide = styled.div`
+  min-width: 100%;
+  transition: 0.5s;
+  img {
+    transition: 0.5s;
+    border-radius: 2px;
+  }
+`
+// left button for carousel
+const LeftButton = styled.button`
+  position: absolute;
+  width: 10%;
+  margin-left: 19%;
+  height: 60%;
+  border: none;
+  background-color: transparent;
+  color: black;
+  cursor: pointer;
+  transition: 0.5s;
+  color: transparent;
+  &:hover {
+    background-color: rgba(255,255,255,0.3);
+    border-radius: 10px;
+    color: black;
+  }
+`
+// right button for carousel
+const RightButton = styled.button`
+  position: absolute;
+  right: 1%;
+  width: 10%;
+  height: 60%;
+  border: none;
+  background-color: transparent;
+  color: black;
+  cursor: pointer;
+  transition: 0.5s;
+  color: transparent;
+  &:hover {
+    background-color: rgba(255,255,255,0.3);
+    border-radius: 10px;
+    color: black;
+  }
+`
+
 export default function OverviewCarousel({ photos }) {
-  var photoList = photos.map((photo) => {
-    return <div key={photo.url}><img src={photo.url}/></div>
-  })
+  // this state is the viewpoint that the carousel is currently at, when the button is pressed, it increases the current viewpoint of the div by 100, thus changing the photo.
+  // Here is a link to the carousel video I watched to get this down. Bad music but good video
+    // https://www.youtube.com/watch?v=Tdpq-9XYoNM
+  var [x, setx] = useState(0)
```

Comment:
This makes sense to me, just curious why adding and subtracting by 100 though?

Reply:
Each picture takes up 100vw, so in order to move completely to the next picture, you have to change it by 100.